web-react/src/pages/directory/quick-facts/QuickFacts.css

```diff
@@ -0,0 +1,158 @@
+:root {
+  --background-base: #121212;
+  --apple-cyan: #0bc9d2;
+  --text-blue: #68e0d7;
+  --lighter-card: #242323;
+  --text-red: #da0218;
+  ---ffffff: #ffffff;
```

Comment:
This is not a good name for a css variable

Reply:
yes just realised, I have removed it

Reply:
yes just realised, I have removed it